packages/transaction-controller/src/TransactionController.ts

```diff
@@ -174,6 +175,8 @@ export class TransactionController extends BaseController<
 > {
   private ethQuery: EthQuery;
 
+  private readonly isAdvancedGasFeeDisabled: boolean;
```

Comment:
We shouldn't need a boolean for this as the callback is optional meaning if the client doesn't want to use it, they simply don't provide the callback.

Reply:
Removed this